component/src/main/java/com/fioalpha/poc/component/view/ResultInvestmentMedium.kt

```diff
@@ -1,33 +1,28 @@
 package com.fioalpha.poc.component.view
 
 import android.content.Context
-import android.text.Editable
-import android.text.TextWatcher
 import android.util.AttributeSet
 import android.view.LayoutInflater
-import android.widget.EditText
 import android.widget.LinearLayout
 import com.fioalpha.poc.component.databinding.InvestmentResultMediumBinding
 import com.fioalpha.poc.component.extension.setMonetary
 import com.fioalpha.poc.component.extension.setMoneyWithTaxes
 import com.fioalpha.poc.component.view.model.ResultInvestmentMediumModel
-import java.text.NumberFormat
 import java.util.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fioalpha/CaseStudy/blob/6ce9cd99c0c1ee323fc1eba2aae62b43356b0111/component/src/main/java/com/fioalpha/poc/component/view/ResultInvestmentMedium.kt#L11"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>